### PR TITLE
Style spec jekyll reorganization and inline examples

### PR DESCRIPTION
**Pull request by bhousel**
- [X] Support inline `example` property and use it to generate jekyll site
- [X] Minify - remove `doc` and `example` properties from `v8.min.json`, `latest.min.json`
- [X] Cleanup `_generate/index.html`
- [X] Fix tests

I'll also pull in most of the other open issues with the 'docs' label.

### CODE REVIEW

**Comment by bhousel** [comment]
@jfirebaugh I'm done with this for now, want to :eyes: ?  

Let me know if it looks ok to bump minor version and merge..

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,29 @@
+0 info it worked if it ends with ok
```
Comment (jfirebaugh) [inline npm-debug.log]: Don't check this in :smiley:

**Comment by jfirebaugh** [comment]
:+1: Looks great other than the `npm-debug.log`!

File: npm-debug.log (hunk 1 above)
Comment (bhousel) [inline npm-debug.log]: haha that thing snuck in on the last commit.. I'll get rid of it and merge.

**Pull request merged**